tests/kafkatest/services/performance/consumer_performance.py

```diff
@@ -70,14 +70,16 @@ class ConsumerPerformanceService(PerformanceService):
             "collect_default": True}
     }
 
-    def __init__(self, context, num_nodes, kafka, topic, messages, version=DEV_BRANCH, new_consumer=True, settings={}):
+    def __init__(self, context, num_nodes, kafka, topic, messages, version=DEV_BRANCH, new_consumer=True, settings={}, config={}, timeout_sec=None):
```

Comment:
Isn't it `settings` param is meant for consumer config? I understand it is from reading the code, although I don't see anyone ever used it yet. Can you please verify so that we don't add the same param but with a different name?

Reply:
Looks like `settings` is meant to specify additional command line options?
```
        for key, value in self.settings.items():
            cmd += " %s=%s" % (str(key), str(value))
```

 I don't think we have a way to specify consumer configs on the command line with `ConsumerPerformance` though which is what `config` is for.

Reply:
Hm.. I tried to run `kafka-consumer-perf-test.sh` and also looked at ConsumerPerformance.scala, does not seem like any of those `" %s=%s"` are actually used... I guess, lets not mess with `settings` in this PR. However, I think it may be better to merge `config` with `self.security_config` and then call `node.account.create_file(ConsumerPerformanceService.CONFIG_FILE, ...)` instead of echo'ing into that file with ssh, especially with one ssh per config item.